server/Pipfile

```diff
@@ -8,6 +8,7 @@ pylint = "*"
 
 [packages]
 flask = "*"
+pymodm = "*"
```

Comment:
Can we as a team start freezing the version numbers so that we don't have version upgrading issues (which has totally happened before 👍

Reply:
Done